fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/flaps_computer.rs

```diff
@@ -210,7 +211,7 @@ impl SlatFlapControlComputer {
                 if context.is_in_flight()
                     && (context.indicated_airspeed().get::<knot>()
                         < Self::ALPHA_SPEED_LOCK_IN_AIRSPEED_THRESHOLD_KNOTS
-                        || context.angle_of_attack().get::<degree>()
+                        || adirs_output.angle_of_attack(1).value().get::<degree>()
```

Comment:
You should check the SSM before using the value. Should it just blindly always use ADR1, or should there be some selection logic to choose an ADR based on validity of the data/SSM?

Reply:
Good point, forgot about the SSM, didn't know how these are handled in the rust libs. 
The selection logic for ADR data should be the same everywhere: 
Median value if alle three operational,
mean if only two operational
don't use it of only one operational

But I haven't seen this selection logic implemented somewhere, and I don't think it makes sense to handle this on every consumer's side (nor do I have the Rust skills to implement a selector). I'll maybe go with the approach seen in air cond, which is chose the first one with valid SSM

Reply:
> The selection logic for ADR data should be the same everywhere:

No, many systems have different logic for specific reasons.

Reply:
just pushed the agreed selection logic (pick first one with valid data)